content/android/_index.md

```diff
@@ -11,4 +11,8 @@ listdescription: ""
 
```

Comment:
The Overview page seems really sparse. I think we should consolidate it with Get Started, keeping the name Get Started and dropping Overview. The opening paragraphs in both seem like they could be condensed slightly and combined.